### PR TITLE
[DO NOT MERGE] feat: Build Pricing Table with Azion Edge SQL

### PR DESCRIPTION
**Pull request by robsongajunior**
## FEAT
- Adding pricing table with Edge SQL during build time
- New Componente: src/components/Table/TablePricing.astro
- Pricing pages mdx

## ISSUE
[UXE-2171](https://aziontech.atlassian.net/browse/UXE-2171)

https://[host]/api/pricing/get/product_slug/content_delivery

[UXE-2171]: https://aziontech.atlassian.net/browse/UXE-2171?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Table/TablePricing.astro [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+} = Astro.props;
+
+const i18n = {
+	"pt-br": {
+		'USA': 'Estados Unidos',
+		'Canada': 'Canadá',
+		'Europe': 'Europa',
+		'Brazil': 'Brasil',
+		'Latam': 'Latam',
+		'Other_Regions': 'Outras_Regioes',
```
Comment (MarianaAguilera) [inline src/components/Table/TablePricing.astro:18]:
```suggestion
		'Other_Regions': 'Outras_Regiões',
```

Comment (MarianaAguilera) [reply src/components/Table/TablePricing.astro:18]: aqui precisa do _ mesmo? @robsongajunior

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 2]
```diff
@@ -0,0 +1,597 @@
...
+    'Latam',
+    'All Other Regions'
+  ]}
+  product_slug="application_acceleration"
+  metric_slug="application_acceleration_data_transferred"
+/>
+
+#### Price example
+
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:92]:
```suggestion
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 3]
```diff
@@ -0,0 +1,597 @@
...
+If your **Application Accelerator** enabled application generated 30TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:93]:
```suggestion
If your **Application Accelerator** enabled application generated 30 TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 4]
```diff
@@ -0,0 +1,597 @@
...
+    'Latam',
+    'All Other Regions'
+  ]}
+  product_slug="content_delivery"
+  metric_slug="cdn_requests"
+/>
+
+#### Price example
+If your applications received 750 million requests from the United States in 1 month,
+consuming a total traffic of 30TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:58]:
```suggestion
If your applications received 750 million requests from the United States in 1 month, consuming a total traffic of 30 TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 5]
```diff
@@ -0,0 +1,597 @@
...
+
+|  | |
+| :--- | ---: |
+| **Total Data Transferred**<br/>The first 10TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
+| The next 20TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
+| In this example, the price for this month will be | 2,450 USD | 
+| The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
+| **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **3,125 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:66]:
```suggestion
| **Total Data Transferred**<br/>The first 10 TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
| The next 20 TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
| In this example, the price for this month will be | 2,450 USD | 
| The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **3,125 USD** | 
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 6]
```diff
@@ -0,0 +1,597 @@
...
+#### Price example
+
+
+If your **Application Accelerator** enabled application generated 30TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
+
+| | |
+| :--- | ---: |
+| The first 10TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
+| The next 20TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
+| **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **1,100 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:99]:
```suggestion
| The first 10 TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
| The next 20 TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **1,100 USD** | 
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 7]
```diff
@@ -0,0 +1,597 @@
...
+
+#### Price example
+
+If your **Load Balancer** enabled application generated 30TB of data transferred in Brazil in 1 month, the price will be calculated as follows:
+
+| | |
+| :--- | ---: |
+| The first 10TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
+| The next 20TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
+| **Total charges**<br/>According to this simulation, the total amount to be paid in this month will be | **1,100 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:223]:
```suggestion
If your **Load Balancer** enabled application generated 30 TB of data transferred in Brazil in 1 month, the price will be calculated as follows:

| | |
| :--- | ---: |
| The first 10 TB of traffic is priced at 0.040 USD per GB, so 10,000 x 0.040 USD | 400 USD | 
| The next 20 TB of traffic is priced at 0.035 USD per GB, so 20,000 x 0.035 USD | 700 USD | 
| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **1,100 USD** |
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 8]
```diff
@@ -0,0 +1,597 @@
...
+| Over 1<br/>TB/month | 0.60 USD |
+
+| Queries | Unitary |
+| --- | ----------- |
+| First 1,000<br/>queries/month | Included |
+| Over 1,000<br/>queries/month | 0.10 USD |
+
+#### Price example
+
+If your applications have 300TB of traffic in a month, most of that volume is data, such as payload, while other traffic is metadata, such as HTTP headers, the request made, status, or user information. Real-Time Events collects some of the metadata, which are made available for your query. The metadata collected represents on average *0.2% of your traffic*.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:454]:
```suggestion
If your applications have 300 TB of traffic in a month, most of that volume is data, such as payload, while other traffic is metadata, such as HTTP headers, the request made, status, or user information. Real-Time Events collects some of the metadata, which are made available for your query. The metadata collected represents on average *0.2% of your traffic*.
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 9]
```diff
@@ -0,0 +1,597 @@
...
+
+| | |
+| :--- | ---: |
+| In this simulation, 600GB (300 TB x 0.2%) will be gathered. |  | 
+| The first 400GB will be charged at a unit price of 1.50 USD, so 400 x 1.50 USD | 600 USD | 
+| The remaining 200GB will be charged at a unit price of 1.20 USD, so 200 x 1.20 USD | 240 USD |
+| **Total charges**<br/>Considering that the limit of 1,000 queries isn't exceeded, the total price calculated in this simulation will be | **840 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:461]:
```suggestion
| In this simulation, 600 GB (300 TB x 0.2%) will be gathered. |  | 
| The first 400 GB will be charged at a unit price of 1.50 USD, so 400 x 1.50 USD | 600 USD | 
| The remaining 200 GB will be charged at a unit price of 1.20 USD, so 200 x 1.20 USD | 240 USD |
| **Total charges**<br/>Considering that the limit of 1,000 queries isn't exceeded, the total price calculated in this simulation will be | **840 USD** |
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 10]
```diff
@@ -0,0 +1,602 @@
...
+    'Latam',
+    'Todas outras regiões'
+  ]}
+  product_slug="content_delivery"
+  metric_slug="cdn_requests"
+/>
+
+#### Exemplo de preço
+
+Se suas aplicações receberam, em um mês, 750 milhões de requests provenientes dos Estados Unidos, consumindo um tráfego total de 30TB, e você ficou dentro do limite de 1.000 purges nesse mês, o valor será calculado da seguinte forma:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:58]:
```suggestion
Se suas aplicações receberam, em um mês, 750 milhões de requests provenientes dos Estados Unidos, consumindo um tráfego total de 30 TB, e você ficou dentro do limite de 1.000 purges nesse mês, o valor será calculado da seguinte forma:
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 11]
```diff
@@ -0,0 +1,602 @@
...
+    'Latam',
+    'Todas outras regiões'
+  ]}
+  product_slug="application_acceleration"
+  metric_slug="application_acceleration_data_transferred"
+/>
+
+#### Exemplo de preço
+
+Se sua aplicação com **Application Accelerator** habilitado gerou, dentro de um mês, 30TB de dados trafegados no Brasil, o valor será calculado da seguinte forma:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:92]:
```suggestion
Se sua aplicação com **Application Accelerator** habilitado gerou, dentro de um mês, 30 TB de dados trafegados no Brasil, o valor será calculado da seguinte forma:
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 12]
```diff
@@ -0,0 +1,602 @@
...
+
+| | |
+| :--- | ---: |
+| **Total Data Transferred**<br/>Os primeiros 10TB trafegados são calculados pelo preço de 0,040 USD por GB, portanto, 10.000 x 0,040 USD | 400 USD | 
+| Os próximos 20TB trafegados são calculados pelo preço de 0,035 USD por GB, portanto | 700 USD | 
+| **Valor total**<br/>De acordo com a simulação. o valor total a ser pago no mês será de | **1.100 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:98]:
```suggestion
| **Total Data Transferred**<br/>Os primeiros 10 TB trafegados são calculados pelo preço de 0,040 USD por GB, portanto, 10.000 x 0,040 USD | 400 USD | 
| Os próximos 20 TB trafegados são calculados pelo preço de 0,035 USD por GB, portanto | 700 USD | 
| **Valor total**<br/>De acordo com a simulação. o valor total a ser pago no mês será de | **1.100 USD** | 
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 13]
```diff
@@ -0,0 +1,602 @@
...
+
+#### Exemplo de preço
+
+Se sua aplicação com **Load Balancer** habilitado gerou dentro de um mês 30TB de dados trafegados no Brasil, o valor será calculado da seguinte forma:
+
+| | |
+| :--- | ---: |
+| **Data Transferred**<br/>Os primeiros 10TB trafegados são calculados pelo preço de 0,040 USD por GB, portanto 10.000 x 0,040 USD | 400 USD | 
+| Os próximos 20TB trafegados são calculados pelo preço de 0,035 USD por GB, portanto 20.000 x 0,035 USD | 700 USD | 
+| **Valor total**<br/>De acordo com a simulação, o valor total a ser pago no mês será de | **1.100 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:224]:
```suggestion
Se sua aplicação com **Load Balancer** habilitado gerou dentro de um mês 30 TB de dados trafegados no Brasil, o valor será calculado da seguinte forma:

| | |
| :--- | ---: |
| **Data Transferred**<br/>Os primeiros 10 TB trafegados são calculados pelo preço de 0,040 USD por GB, portanto 10.000 x 0,040 USD | 400 USD | 
| Os próximos 20 TB trafegados são calculados pelo preço de 0,035 USD por GB, portanto 20.000 x 0,035 USD | 700 USD | 
| **Valor total**<br/>De acordo com a simulação, o valor total a ser pago no mês será de | **1.100 USD** |
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 14]
```diff
@@ -0,0 +1,602 @@
...
+Free of charge
+</Badge>
+
+Mitigue os maiores e mais complexos ataques DDoS nas camadas de rede e aplicação.
+
+<Button href="https://www.azion.com/pt-br/produtos/ddos-protection/" text="Acesse os detalhes do produto" variant="secondary"></Button>
+
+### Network Layer Protection
+
+Um perímetro de segurança programável no edge para tráfego de entrada e saída
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:242]:
```suggestion
Um perímetro de segurança programável no edge para tráfego de entrada e saída.
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 15]
```diff
@@ -0,0 +1,602 @@
...
+
+#### Exemplo de preço
+
+Se suas aplicações trafegam 300TB em um mês, uma grande parte desse volume é composta por dados, como o payload, e parte é composta por metadados, tais como cabeçalhos HTTP, a requisição realizada, o status ou informações sobre seu usuário. O **Real-Time Events** coleta alguns dos metadados, que ficam disponíveis para sua consulta. Os metadados coletados representam, em média, *0,2% de seu tráfego*.
+| | |
+| :--- | ---: |
+| Nessa simulação, serão coletados 600 GB (300 TB x 0,2%) |  | 
+| Os primeiros 400GB serão tarifados pelo preço unitário de 1,50 USD, ou seja, 400 x 1,50 USD | 600 USD | 
+| Os 200GB restantes serão tarifados pelo preço unitário de 1,20 USD, ou seja, 200 x 1,20 USD | 240 USD |
+| **Valor total**<br/>Considerando que não foram excedidas as 1.000 consultas no mês, o valor total apurado, nessa simulação, será de| **840 USD** |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:467]:
```suggestion
Se suas aplicações trafegam 300 TB em um mês, uma grande parte desse volume é composta por dados, como o payload, e parte é composta por metadados, tais como cabeçalhos HTTP, a requisição realizada, o status ou informações sobre seu usuário. O **Real-Time Events** coleta alguns dos metadados, que ficam disponíveis para sua consulta. Os metadados coletados representam, em média, *0,2% de seu tráfego*.
| | |
| :--- | ---: |
| Nessa simulação, serão coletados 600 GB (300 TB x 0,2%) |  | 
| Os primeiros 400 GB serão tarifados pelo preço unitário de 1,50 USD, ou seja, 400 x 1,50 USD | 600 USD | 
| Os 200 GB restantes serão tarifados pelo preço unitário de 1,20 USD, ou seja, 200 x 1,20 USD | 240 USD |
| **Valor total**<br/>Considerando que não foram excedidas as 1.000 consultas no mês, o valor total apurado, nessa simulação, será de| **840 USD** |
```

File: src/components/Table/TablePricing.astro (hunk 1 above)
Comment (robsongajunior) [reply src/components/Table/TablePricing.astro:18]: precisa

File: src/components/Table/TablePricing.astro [hunk 16]
```diff
@@ -0,0 +1,157 @@
...
+
+		'First 10 million images processed/month': 'Primeiros 10 milhões de imagens processadas/mês',
+		'Next 40 million images processed/month': 'Próximos 40 milhões de imagens processadas/mês',
+		'Next 100 million images processed/month': 'Próximos 100 milhões de imagens processadas/mês',
+		'Next 350 million images processed/month': 'Próximos 350 milhões de imagens processadas/mês',
+		'Next 500 million images processed/month': 'Próximos 500 milhões de imagens processadas/mês',
+		'Next 4,000 million images processed/month': 'Próximos 4.000 milhões de imagens processadas/mês',
+		'Over 5,000 million images processed/month': 'Acima de 5.000 milhões de imagens processadas/mês',
+
+		'First 1 billion requests/month': 'Primeiros 1 bilhão de requisições/mês',
```
Comment (MarianaAguilera) [inline src/components/Table/TablePricing.astro:42]:
```suggestion
		'First 1 billion requests/month': 'Primeiro 1 bilhão de requisições/mês',
```

File: src/components/Table/TablePricing.astro [hunk 17]
```diff
@@ -0,0 +1,157 @@
...
+		'First 1 billion requests / month': 'Primeiros 1 bilhão de requisições / mês',
```
Comment (MarianaAguilera) [inline src/components/Table/TablePricing.astro:43]:
```suggestion
		'First 1 billion requests / month': 'Primeiro 1 bilhão de requisições / mês',
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 18]
```diff
@@ -0,0 +1,595 @@
...
+If your applications received 750 million requests from the United States in 1 month, consuming a total traffic of 30 TB, and you were within the limit of 1,000 purges in that month, the price will be calculated as follows:
+
+|  | |
+| :--- | ---: |
+| **Total Data Transferred**<br/>The first 10 TB of traffic is priced at 0.085 USD per GB, so 10,000 x 0.085 USD | 850 USD | 
+| The next 20 TB of traffic is priced at 0.080 USD per GB, so 20,000 x 0.080 USD | 1,600 USD | 
+| In this example, the price for this month will be | 2,450 USD | 
+| The charge is for every 10,000 requests, at a unit price of 0.009 USD, so (750,000,000 / 10,000) x 0.009 USD | 675 USD | 
+| **Total charges**<br/>According to this simulation, the total amount to be paid this month will be | **3,125 USD** | 
+
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:66]:
```suggestion
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 19]
```diff
@@ -0,0 +1,595 @@
...
+
+### Application Accelerator
+
+Speed ​​up the performance of your applications and APIs with protocol optimizations and advanced request and response rules.
+
+<Button href="https://www.azion.com/en/products/application-acceleration/" text="Access product details" variant="secondary"></Button>
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:72]:
```suggestion
<Button href="https://www.azion.com/en/products/application-accelerator/" text="Access product details" variant="secondary"></Button>
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 20]
```diff
@@ -0,0 +1,595 @@
...
+    'Price per GB'
+  ]}
+  product_slug="data_streaming"
+  metric_slug="data_streaming_data_streamed"
+/>
+
+
+#### Price example
+
+If you had 3 billion **Data Stream** requests and streamed 25TB of data, the value would be calculated as follows:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:400]:
```suggestion
If you had 3 billion **Data Stream** requests and streamed 25 TB of data, the value would be calculated as follows:
```

File: src/content/docs/en/pages/main-menu/pricing/pricing.mdx [hunk 21]
```diff
@@ -0,0 +1,595 @@
...
+This is the total number of **Real-Time Events** queries. The first 1,000 queries in a month are included for all clients. When exceeding this volume, you'll be charged 0.10 USD per query.
+:::
+
+---
+
+## Azion Service Plans
+
+### Developer Plan
+
+For developers who want to explore the potential of our platform themselves.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/pricing/pricing.mdx:470]:
```suggestion
For developers who want to explore the potential of Azion's platform themselves.
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 22]
```diff
@@ -0,0 +1,602 @@
...
+
+| | |
+| :--- | ---: |
+| **Tempo de computação**<br/>(5.000.000 de invocações x 0,030 segundos / 1.000) x 0,049 USD | 7,35 USD | 
+| **Invocations**<br/>(5.000.000 / 1.000.000) x 0,60 USD | 3 USD | 
+| **Valor total**<br/>No exemplo, o total tarifado é | **10,35 USD** | 
+
+### Image Processor
+
+Otimização e manipulação de imagens em tempo real para aprimorar a experiência do usuário
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:168]:
```suggestion
Otimização e manipulação de imagens em tempo real para aprimorar a experiência do usuário.
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 23]
```diff
@@ -0,0 +1,602 @@
...
+#### Exemplo de preço
+
+Se você hospedou 30 zonas no **Edge DNS** e recebeu 1,5 bilhão de queries ao longo do mês, o valor será calculado da seguinte forma:
+
+| | |
+| :--- | ---: |
+| **Hosted Zones**<br/>As primeiras 25 zonas são tarifadas ao preço unitário de 25 USD, portanto | 625 USD | 
+| As 5 zonas restantes são tarifadas ao preço unitário de 15 USD, portanto | 75 USD | 
+| Nesse exemplo, as 30 zonas custarão | 700 USD |
+| **Standard Queries**<br/>The first billion queries will be priced at 0.85 USD por milhão. Therefore (1,000,000,000 / 1,000,000) x 0.85 USD | 850 USD |
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:359]:
```suggestion
| **Standard Queries**<br/>O primeiro bilhão de queries será tarifado ao preço de 0,85 USD por milhão. Portanto (1.000.000.000 / 1.000.000) x 0,85 USD | 850 USD | 
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 24]
```diff
@@ -0,0 +1,602 @@
...
+| --- | ----------- |
+| Primeiros 400<br/>GB/mês | 1,50 USD |
+| Próximos 600<br/>GB/mês | 1,20 USD |
+| Acima de 1<br/>TB/mês | 0,60 USD |
+
+| Queries | Unitário |
+| --- | ----------- |
+| Primeiras 1.000<br/>queries/mês | Incluso |
+| Acima de 1.000<br/>queries/mês | 0,10 USD |
+
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:457]:
```suggestion
```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 25]
```diff
@@ -0,0 +1,602 @@
...
+
+#### Exemplo de preço
+
+Se suas aplicações trafegam 300 TB em um mês, uma grande parte desse volume é composta por dados, como o payload, e parte é composta por metadados, tais como cabeçalhos HTTP, a requisição realizada, o status ou informações sobre seu usuário. O **Real-Time Events** coleta alguns dos metadados, que ficam disponíveis para sua consulta. Os metadados coletados representam, em média, *0,2% de seu tráfego*.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:461]:
```suggestion
Se suas aplicações trafegam 300 TB em um mês, uma grande parte desse volume é composta por dados, como o payload, e parte é composta por metadados, tais como cabeçalhos HTTP, a requisição realizada, o status ou informações sobre seu usuário. O **Real-Time Events** coleta alguns dos metadados, que ficam disponíveis para sua consulta. Os metadados coletados representam, em média, *0,2% de seu tráfego*.

```

File: src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx [hunk 26]
```diff
@@ -0,0 +1,602 @@
...
+Trata-se do total de consultas que você realiza no **Real-Time Events**. As primeiras 1.000 consultas por mês estão inclusas para todos os clientes. A partir desse volume, serão tarifados 0,10 USD por consulta.
+:::
+
+---
+
+## Planos de Serviço da Azion
+
+### Developer Plan
+
+Grátis. Para desenvolvedores que se sentem confortáveis em explorar o potencial de nossa plataforma com autonomia.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/menu-principal/pricing/pricing.mdx:479]:
```suggestion
Grátis. Para desenvolvedores que se sentem confortáveis em explorar o potencial da plataforma da Azion com autonomia.
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**